hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieBootstrapConfig.java

```diff
@@ -107,8 +105,80 @@
       .defaultValue(HFileBootstrapIndex.class.getName())
       .sinceVersion("0.6.0")
```

Comment:
Rename to `BOOTSTRAP_INDEX_CLASS_NAME` ? Since we are changing all `_CLASS` to `_CLASS_NAME` ?

Reply:
done.